docs/source/kedro_project_setup/dependencies.md

```diff
@@ -4,45 +4,22 @@ Both `pip install kedro` and `conda install -c conda-forge kedro` install the co
 
 When you create a project, you then introduce additional dependencies for the tasks it performs.
 
-## Project-specific dependencies
+## Declare project-specific dependencies
 
-You can specify a project's exact dependencies in the `requirements.txt` file to make it easier for you and others to run your project in the future,
-and to avoid version conflicts downstream. This can be achieved with the help of [`pip-tools`](https://pypi.org/project/pip-tools/).
-To install `pip-tools` in your virtual environment, run the following command:
-
-```bash
-pip install pip-tools
-```
-
-To add or remove dependencies to a project, edit the `requirements.txt` file, then run the following:
-
-```bash
-pip-compile <project_root>/requirements.txt --output-file <project_root>/requirements.lock
-```
-
-This will [pip compile](https://github.com/jazzband/pip-tools#example-usage-for-pip-compile) the requirements listed in
-the `requirements.txt` file into a `requirements.lock` that specifies a list of pinned project dependencies
-(those with a strict version). You can also use this command with additional CLI arguments such as `--generate-hashes`
-to use `pip`'s Hash Checking Mode or `--upgrade-package` to update specific packages to the latest or specific versions.
-[Check out the `pip-tools` documentation](https://pypi.org/project/pip-tools/) for more information.
-
-```{note}
-The `requirements.txt` file contains "source" requirements, while `src/requirements.lock` contains the compiled version of those and requires no manual updates.
-```
+You can specify a project's exact dependencies in the `requirements.txt` file to make it easier for you and others to run your project in the future, and to avoid version conflicts downstream. When you create a new Kedro project, a `requirements.txt` file is generated in the root directory of the project. This file contains the core dependencies and dependecies related to the tools chosen during the project creation process.
```

Comment:
```suggestion
You can specify a project's exact dependencies in the `requirements.txt` file to make it easier for you and others to run your project in the future, and to avoid version conflicts downstream. When you create a new Kedro project, a `requirements.txt` file is generated in the root directory of the project. This file contains the core dependencies and dependencies related to the tools chosen during the project creation process.
```